src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppresswithnearbytextfilter/InputSuppressWithNearbyTextFilterNearbyTextPattern.css

```diff
@@ -0,0 +1,58 @@
+/*
+SuppressWithNearbyTextFilter
+nearbyTextPattern = cs (\\w+)
```

Comment:
make it `-cs:` it is more clear and common pattern already

Reply:
Done.